### DISCUSSION TITLE
Steps Reverse

### DISCUSSION THREAD

**Discussion by malawllaqi** [ant-design/ant-design#41478] [Q&A]
Hi,

I'm Building a page for shipment tracking and I use the Steps Component for it.

How can I reverse the steps component, I want the last step item to be first.

<img width="375" alt="steppers" src="https://user-images.githubusercontent.com/85809249/227880311-3a447d20-e2fc-41a4-bdcc-29b13c9f95e5.png">

**Comment by MuxinFeng** [comment]
maybe you can reverse data from the server before passing values ​​to components.
